### PR TITLE
Nuke resprite revival

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Resprites the nuke and adds new visuals for the core being stolen. Revival of #20886 with code added.

## Why It's Good For The Game
old sprite so bad, as per the old PR

## Images of changes

image
image

https://user-images.githubusercontent.com/91113370/232622417-08513514-9b6c-43ef-96fa-1fd6ec3ec94f.mp4

## Testing

See above, plus tested syndicate and deathsquad nukes. Discovered a lot of things, but no logic was changed so... "working as intended".

Also tried closing the panel/unanchoring the nuke via tools while hacking into the core. Also put the core back in the nuke while doing this, the visuals stayed as expected.

## Changelog
:cl: McRamon, Contrabang
imageadd: New sprites for the nuke
imageadd: New sprites for the nuke core being stolen
/:cl:

### CODE REVIEW

**Comment by McRamon** [comment]
image
Note for smart guys - i am giving the heart for the code i was unable to do myself, not the sprites i made 🔫

**Comment by Contrabang** [comment]
This still needs a sprite replacement for when blob captures the nuke, gonna put it on a draft until that is done

**Comment by Contrabang** [comment]
Blob nuke added 
image

**Comment by Sirryan2002** [comment]
Don't like the clashing between the blob-nuke and nuke core, but that's probably due to blob needing a resprite because the new sprite being proposed in this PR is much better.

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 1]
```diff
@@ -115,7 +124,25 @@ GLOBAL_VAR(bomb_set)
...
+	switch(selected_stage)
+		if(NUKE_CORE_PANEL_EXPOSED)
+			. += "nukecore1"
+		if(NUKE_CORE_PANEL_UNWELDED)
+			. += "nukecore2"
+		if(NUKE_CORE_FULLY_EXPOSED)
+			if(core)
+				. += "nukecore3"
+			else
+				. += "nukecore4"
```
Comment (Sirryan2002) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:145]:
```suggestion
			. += core ? "nukecore3" : "nukecore4"
```

**Review by Sirryan2002 — APPROVED** [review]

**Pull request merged**